### DISCUSSION TITLE
Codespace disappearing?

### DISCUSSION THREAD

**Discussion by beatmakerofficial** [community/community#110454] [Codespaces]
### Select Topic Area

General

### Body

I was given a Codespace to do work in for a computer science class. I completed the course and started a new one but kept using the same codespace to store all my work. Now, all of a sudden, I can't access the Codespace anymore. I contacted the class and they said to check for emails from Github about the codespace being closed, but I didn't have any. Can someone help please?

**Comment by VladimirCreator** [comment]
You need manually disable automatic deletion of your Codespace.
